File: GenerateCode/GenerateCode/bin/Debug/DeviceBrandPMSetting/PPPWeb/app/ViewModels/DeviceBrandPMSetting/SearchListViewModel.ts
//#region Common Import									
import { PageViewModel } from '../BaseViewModel';		 
//#endregion											 
														 
//#region Enum                                           
export enum EnumSort                                     
{                                                        
    DEFAULT = ""                                       
    // , PARAMETER_CATEGORY_CD_ASC = "PARAMETER_CATEGORY_CD_ASC" 
    // , PARAMETER_CATEGORY_CD_DESC = "PARAMETER_CATEGORY_CD_DESC" 
}                                                        
//#endregion                                             
//#region Class											 
export class SearchInfoViewModel extends PageViewModel  {
	Sort: EnumSort = null;  							 
														 
}														 
														 
export class SearchListViewModel extends PageViewModel { 
    SearchItemList: SearchItemViewModel[]= [];			 
}														 
														 
export class SearchItemViewModel extends PageViewModel { 
														 
}														 
//#endregion											 

File: GenerateCode/GenerateCode/bin/Debug/HandsetSellPriceHistory/PPPWebAPI/Models/ViewModels/HandsetSellPriceHistory/SearchListViewModel.cs
using System;                                            
using System.Collections.Generic;						 
using System.ComponentModel;							 
using System.Linq;										 
using System.Web;										 
using PPPWEBAPI.Extensions;								 
														 
namespace PPPWEBAPI.Models.ViewModels.HandsetSellPriceHistory 
{														 
	/// <summary>                                                    
    /// 查詢結果													 
    /// </summary>													 
    public class SearchListViewModel : _BaseSearchListViewModel		 
    {																 
        #region Properties											 
        /// <summary>												 
        /// 查詢結果清單											   
        /// </summary>												 
        public List<SearchItemViewModel> SearchItemList { get; set; }
																	 
        #endregion													 
    }																 
																	 
    /// <summary>													 
    /// 查詢結果細項												   
    /// </summary>													 
    public class SearchItemViewModel : _BaseViewModel				 
    {																 
        #region Properties											 
																	 
        #endregion													 
    }																 
}														 
